softlayer/vm/softlayer_creator.go

```diff
@@ -67,9 +74,43 @@ func (c SoftLayerCreator) Create(agentID string, stemcell bslcstem.Stemcell, clo
 	}
 
 	agentEnvService := c.agentEnvServiceFactory.New(virtualGuest.Id)
-
 	vm := NewSoftLayerVM(virtualGuest.Id, c.softLayerClient, util.GetSshClient(), agentEnvService, c.logger, TIMEOUT_TRANSACTIONS_DELETE_VM)
 
+	if len(cloudProps.BoshIp) == 0 {
```

Comment:
Where is the additional unit tests for the new behavior added from lines 79-113?